dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/pql/convert/FiltersConverter.java

```diff
@@ -418,14 +451,14 @@ private static boolean isNestedField(@NonNull JqlField field) {
     // The fields must be correctly separated before the next steps.
     val isMutation = (indexType == MUTATION_CENTRIC) && typePrefix.equals("mutation");
     if (isMutation) {
-      final Predicate<JqlField> nestedPredicate = (f) -> isNestedField(f);
-      final Predicate<JqlField> notNestedPredicate = (f) -> !isNestedField(f);
+      final Predicate<JqlField> nestedPred = (f) -> isNestedField(f);
+      final Predicate<JqlField> notNestedPred = (f) -> !isNestedField(f);
 
-      val nonNestedFields = fields.stream().filter(notNestedPredicate).collect(toList());
+      val nonNestedFields = fields.stream().filter(notNestedPred).collect(toList());
```

Comment:
Might be more efficient and less code to use `partitioningBy` above:

http://stackoverflow.com/questions/27993604/whats-the-purpose-of-partitioningby

For example:

```java
val partitions = fields.stream().collect(partitioningBy(this:: isNestedField));
```

Prevents having to run through the list twice and the need for the predicates above.